charts/argo-rollouts/templates/controller/deployment.yaml

```
@@ -65,6 +65,10 @@ spec:
       {{- with .Values.controller.extraContainers }}
         {{- toYaml . | nindent 6 }}
       {{- end }}
+      {{- if .Values.controller.initContainers }}
+      initContainers:
+      {{- toYaml .Values.controller.initContainers | nindent 8 }}
+      {{- end }}
```

Let's use the newer syntax :D

```suggestion
      {{- with .Values.controller.initContainers }}
      initContainers:
      {{- toYaml . | nindent 8 }}
      {{- end }}
```